src/server/utils/processManager.js

```diff
@@ -179,7 +177,9 @@ export default class ProcessManager {
     if (!this.initCommand) return;
 
     const env = Object.assign({}, process.env);
-    env.Path += (isWin ? ';' : ':') + process.cwd() + '/node_modules/.bin'; // Using process.cwd until a better solution is found for (#5)
+    const pathVariableName = getPathVariableName();
+    env[pathVariableName] +=
+      (isWin ? ';' : ':') + process.cwd() + '/node_modules/.bin'; // Using process.cwd until a better solution is found for (#5)
```

Comment:
Could we move this comment onto it's own line? I'm not a huge fan of inline comments and this is a long line to read because of it.
Is it possible to update prettier to not allow inline comments?

Reply:
😅 no problem, don't know how to make prettier disallow new lines though

Reply:
No probs, I've just updated the eslint config to disallow inline comments, we'll see how long it lasts! 😉